packages/go_router_builder/CHANGELOG.md

```diff
@@ -1,23 +1,27 @@
+## 1.0.6
+
+* Use path-based deps in example.
```

Comment:
[This comment](https://github.com/flutter/packages/pull/2252#discussion_r904127522) is still unresolved.